docs/ref/models/fields.txt

```diff
@@ -1212,63 +1212,13 @@ define the details of how the relation works.
         edge cases multiple times, so there is a possibility your callable may
         be invoked several times.
 
-.. attribute:: ForeignKey.related_name
-
-    The name to use for the relation from the related object back to this one.
-    It's also the default value for :attr:`related_query_name` (the name to use
-    for the reverse filter name from the target model). See the :ref:`related
-    objects documentation <backwards-related-objects>` for a full explanation
-    and example. Note that you must set this value when defining relations on
-    :ref:`abstract models <abstract-base-classes>`; and when you do so
-    :ref:`some special syntax <abstract-related-name>` is available.
-
-    If you'd prefer Django not to create a backwards relation, set
-    ``related_name`` to ``'+'`` or end it with ``'+'``. For example, this will
-    ensure that the ``User`` model won't have a backwards relation to this
-    model::
-
-        user = models.ForeignKey(User, related_name='+')
-
-.. attribute:: ForeignKey.related_query_name
-
-    The name to use for the reverse filter name from the target model.
-    Defaults to the value of :attr:`related_name` if it is set, otherwise it
-    defaults to the name of the model::
-
-        # Declare the ForeignKey with related_query_name
-        class Tag(models.Model):
-            article = models.ForeignKey(Article, related_name="tags", related_query_name="tag")
-            name = models.CharField(max_length=255)
-
-        # That's now the name of the reverse filter
-        Article.objects.filter(tag__name="important")
-
-.. attribute:: ForeignKey.to_field
-
-    The field on the related object that the relation is to. By default, Django
-    uses the primary key of the related object.
-
-.. attribute:: ForeignKey.db_constraint
-
-    Controls whether or not a constraint should be created in the database for
-    this foreign key. The default is ``True``, and that's almost certainly what
-    you want; setting this to ``False`` can be very bad for data integrity.
-    That said, here are some scenarios where you might want to do this:
-
-    * You have legacy data that is not valid.
-    * You're sharding your database.
-
-    If this is set to ``False``, accessing a related object that doesn't exist
-    will raise its ``DoesNotExist`` exception.
-
 .. attribute:: ForeignKey.on_delete
```

Comment:
Need to updated the sentence ":class:`ForeignKey` accepts an extra set of arguments -- all optional". Probably `on_delete` should be moved so it's first. Also add a `.. versionchanged:: 1.9` annotation saying that it wasn't required in older versions.